src/System.Management.Automation/namespaces/LocationGlobber.cs

```diff
@@ -189,17 +189,12 @@ internal LocationGlobber(SessionState sessionState)
                 TraceFilters(context);
 
                 // First check to see if the path starts with a ~ (home)
-
-                if (IsHomePath(path))
+                using (s_pathResolutionTracer.TraceScope("Resolving HOME relative path."))
                 {
-                    using (s_pathResolutionTracer.TraceScope("Resolving HOME relative path."))
-                    {
-                        path = GetHomeRelativePath(path);
-                    }
+                    path = GetHomeRelativePath(path);
```

Comment:
If we're no longer performing the check, we should update this comment.

Reply:
I left this comment as a clarification of intentions.